certbot/tests/main_test.py

```diff
@@ -114,15 +121,20 @@ def _call(self, args):
             cli.prepare_and_parse_args(plugins, args))
 
         with mock.patch('certbot.main._init_le_client') as mock_init:
-            main.obtain_cert(config, plugins)
+            with mock.patch('certbot.main._suggest_donation'):
+                main.certonly(config, plugins)
 
         return mock_init()  # returns the client
 
-    @mock.patch('certbot.main._auth_from_available')
-    def test_no_reinstall_text_pause(self, mock_auth):
+    @mock.patch('certbot.main._find_cert')
+    @mock.patch('certbot.main._get_and_save_cert')
+    @mock.patch('certbot.main._report_new_cert')
+    def test_no_reinstall_text_pause(self, unused_report, mock_auth,
+        mock_find_cert):
         mock_notification = self.mock_get_utility().notification
         mock_notification.side_effect = self._assert_no_pause
-        mock_auth.return_value = ('reinstall', mock.ANY)
+        mock_auth.return_value = mock.Mock()
+        mock_find_cert.return_value = None, None
```

Comment:
nit: `_find_cert` will never return `None` here. `False` should be used instead.